aplib.net-demo/Assets/Scripts/WFC/Cell.cs

```diff
@@ -37,7 +37,7 @@ public class Cell
         /// <summary>
         /// Gets or sets a value indicating whether this cell contains an item.
         /// </summary>
-        public bool ContainsItem { get; set; }
+        public bool CannotHoldAnItem { get; set; }
```

Comment:
This name is also wrong, for it is set to true once it holds an item. If it holds one, it cannot hold an item? I would say `ConnotHoldAnotherItem` for that reason.